components/sensor/bme280/bme280.go

```diff
@@ -81,18 +84,13 @@ const (
 
 // Config is used for converting config attributes.
 type Config struct {
-	Board   string `json:"board"`
 	I2CBus  string `json:"i2c_bus"`
```

Comment:
This line, though unmodified, has changed semantically. The value used to be something like `"main_i2c_bus"` and is now something like `"2"`.

Reply:
maybe would be helpful to include a comment here that even though it's a string it should be a number?

Reply:
Great idea! It's a string because it's _possible_ for it to be non-numeric, but you're right that 99.9% of the time it'll be a number.